### PR TITLE
21-0972 - Fix typo

### PR DESCRIPTION
**Pull request by tlei123**
## Summary

- Fix link-text typo in static-pages Flipper-widget for Alternate Signer (21-0972)
- Team: Veteran Facing Forms
- Flipper end-date TBD

## Related issue(s)

- _Link to ticket created in va.gov-team-forms repo_
department-of-veterans-affairs/va.gov-team-forms#637

## Testing done

- N/A &dash; just a link-text change

## What areas of the site does it impact?

This form's About page ONLY.

### CODE REVIEW

File: src/applications/static-pages/simple-forms/21-0972/App.js
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/static-pages/simple-forms/21-0972/App.js:1]: @rmessina1010 , this spec is straight from the PM who opened the [ticket](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/637):
image

[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/static-pages/simple-forms/21-0972/App.js:1]: Update, @rmessina1010 !  I just heard back from PM, and changed "...Request.." to "...request...".  Change just pushed.  :)

**Comment by tlei123** [comment]
@rmessina1010 , this is ready for re-review & approval now &mdash; we've made it lowercase ("...request...").  Thanks much in advance.

**Review by rhasselle-oddball — APPROVED** [review]

**Review by CBonade — APPROVED** [review]
LGTM

**Pull request merged**